ngraph/python/tests/test_onnx/test_zoo_models.py

```diff
@@ -616,6 +616,7 @@
 
 xfail_issue_36537(test_cases.test_shufflenet_v2_opset10_cpu)
 xfail_issue_36538(test_cases.test_yolov3_opset10_cpu)
+xfail_issue_36538(test_cases.test_tiny_yolov3_opset11_cpu)
```

Comment:
Why did you need to add this xfail?

Reply:
@postrational  After enabling `Round` the next problem occurs. It is the same exception as for `yolov3_opset10_cpu` - there is created bug for it.